pkgs/data/fonts/nerdfonts-patcher/default.nix

```diff
@@ -0,0 +1,52 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, python3
+, fontforge
+
+, patchFont ? null
+, fontName ? null
+, glyphFlags ? "-c"
+}:
+
+assert (patchFont != null);
+assert (fontName != null);
+
+stdenv.mkDerivation rec {
+  pname = fontName + "-nerd-font";
+  version = "v2.1.0";
+  
+  src = fetchFromGitHub {
```

Comment:
The tarball has a size of 2.2GiB and I think only the file font-patcher needed. Could you try to pick only that file.

Reply:
I have tried this before, and it doesn't work. I think it comes down to the icon sets needing to be downloaded as well.

Reply:
There is already a related issue, could you please add a link to https://github.com/ryanoasis/nerd-fonts/issues/484